src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
The idea simply is that just like captures in standard chess, forcing a capture in giveaway chess can complicate matters, so such moves should not be reduced too much. In my opinion you are overthinking this a bit, since we have applied more radical changes without such a discussion. (By the way, that is what I meant by arbitrariness earlier. I start to understand, but still am not fully able to relate to why you treat this PR so differently than others. That is why I prefer to judge based on more objective measures like test results.)

What do you mean by destabilizing? If you are afraid that a patch does not scale well, you could simply ask for or submit a test at longer TC to clarify it, which I would always consider justified for changes to LMR, extensions, or the like. If it is about search explosions, you could run the search on positions that might trigger such issues and check the results. Or which kind of stability are you referring to? Any I can come up with could rather easily be verified/falsified by testing (running matches or search) without having to deeply think and speculate about it.